parsers/promotedTitle.js

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env nodejs
+var _ = require('lodash');
+var cheerio = require('cheerio');
+var moment = require('moment');
+var debug = require('debug')('promotedInfo');
+var parse = require('./lib/parse');
+var entities = require('entities');
+
+var postcount = 0;
+
+function tryPage(elem) {
+    try {
+        return elem.attr().onmouseover.replace(/.*\"http/, 'http').replace(/\".*/, '').replace(/\\/g, '');
+    } catch(error) {
+        return null;
+    };
+}
+function getPromotedTitle(snippet) {
+
+	var title;
+	var title_type;
+	var error = 0;
+	
+	var e_threshold;
+	var e_ptr;
+		
+	postcount ++;
+	
+    var $ = cheerio.load(snippet.html);
+    
+	var e_threshold = $('div.userContent');
+
+	e_ptr = ($(e_threshold).next().find(".clearfix"));
+	$(e_ptr).find("*").each(function(e) {
+		if (title == undefined) {
+			var re = /(div|a|span)/g;
+			if (re.exec($(this)[0].name) != null) {
+				if ($(this).text() === entities.decodeHTML($(this).first().html()) && $(this).text() !== "") {
+					if ($(this).text() !== "Click for more") {
```

Comment:
@nolash I'm afraid this is wrong, you can't assume all the facebook users are using English layout... for person with other languages, the text appear different. we've to relay, as best as we can, on W3C CSS selectors